qna/DE-custom-datacenter-hosting.md

```diff
@@ -0,0 +1,65 @@
+|                                                              |                        | Einrichtung der  Managementverbindung                        |                           |                               |
+|                                                              |                        | Einbindung der  fiskaltrust.portal Verbindung                |                           |                               |
+|                                                              |                        | Provisionierung der  Basis PODs                              |                           |                               |
+|                                                              |                        | Tests und Freigabe                                           |                           |                               |
+|                                                              |                        |                                                              |                           |                               |
+|                                                              |                        | **Management / SLA 99,0% Jährlich**                          |                           | monatliche Kosten             |
+|                                                              |                        | Monitoring                                                   |                           |                               |
+|                                                              |                        | Einspielen von Patches                                       |                           |                               |
+|                                                              |                        | Updates von Kubernetes  aktivieren (mind. vierteljährlich)   |                           |                               |
+|                                                              |                        | Beheben von Deployment  issues                               |                           |                               |
+|                                                              |                        | Support                                                      |                           |                               |
+|                                                              |                        | Telefon Hotline Mo-Fr  09:00h-15:00h                         |                           |                               |
+|                                                              |                        | Störungsmeldung online per  Formular                         |                           |                               |
+|                                                              |                        | Reaktionszeit: 4h                                            |                           |                               |
```

Comment:
Gesetzlich darf eine Kassa auch im "ausgefallenen" Zustand betrieben werden. Daher ist 4h möglicherweise zu kurz? Bzw. nur in einem Zeitfenster?

Reply:
Ich werde das aufnehmen & intern diskutieren.

Reply:
- Kubernetes Cluster Betrieb liegt beim Kunden oder extern
- Container Betrieb wird ausgelagert auf ACP
- SLA ist angepasst an ACP SLA